ros_ign_bridge/include/ros_ign_bridge/convert.hpp

```diff
@@ -44,10 +44,10 @@
 #include <std_msgs/String.h>
 #include <tf2_msgs/TFMessage.h>
 
-// include Ignition builtin messages
+// include Ignition messages
 #include <ignition/msgs.hh>
 
-#include "ros_ign_bridge/convert_decl.hpp"
+#include <ros_ign_bridge/convert_decl.hpp>
```

Comment:
Not sure about the style. Are we using `<...>` for headers in the same package?

Reply:
good point: acbe7d78fa23145ede636b58edbf89d7c5def704